### PR TITLE
fix Elasticsearch MaxDbStatementLength option

### PR DESCRIPTION
**Pull request by chad122**
Fixes #.

## Changes

Fix Elasticsearch MaxDbStatementLength option

### CODE REVIEW

**Comment by reyang** [comment]
@open-telemetry/dotnet-contrib-maintainers who should own this component? It seems the https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/a2244668374150b1f95e364e83b44f9958c97313/.github/component_owners.yml#L27 configuration is not working with the automatic PR assignment.

**Comment by reyang** [comment]
Followed up in #429.

**Review by ejsmith — APPROVED** [review]

**Comment by reyang** [comment]
@chad122 please update the changelog.

**Review by reyang — APPROVED** [review]

File: src/OpenTelemetry.Instrumentation.ElasticsearchClient/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,7 @@
 * Requests that get an HTTP status code of 404 are not marked as an error span status
 * Add MaxDbStatementLength option with default of 4096
 * Remove duplicated HTTP method and URL from db.statement attribute value
+* Fix faulty logic of MaxDbStatementLength option
```
Comment (Kielek) [inline src/OpenTelemetry.Instrumentation.ElasticsearchClient/CHANGELOG.md:10]:
```suggestion
* Fix faulty logic of MaxDbStatementLength option [(#425)](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/425)
```

**Review by cijothomas — APPROVED** [review]

**Pull request merged**